x-pack/plugins/security/server/config.ts

```diff
@@ -226,6 +226,11 @@ export const ConfigSchema = schema.object({
         }
       },
     }),
+    concurrentSessions: schema.maybe(
+      schema.object({
+        maxSessions: schema.number({ min: 1, max: 1000 }),
```

Comment:
__note:__ I cannot think of any valid use case when a very large limit would be required (even 1000 is unlikely to be useful), so I'm introducing this semi-reasonable limit just to avoid any performance problems in the cleanup routine.